app/hypnos/prj.conf

```diff
@@ -49,3 +49,5 @@ CONFIG_BT_ATT_ENFORCE_FLOW=n
 
 # Power management
 CONFIG_DEVICE_POWER_MANAGEMENT=y
+CONFIG_SYS_PM_STATE_LOCK=y
+CONFIG_SYS_POWER_DEEP_SLEEP_STATES=y
```

Comment:
Remove these two. They don't belong to a cleanup PR.

Reply:
woups. Sunday morning is not the best time to think straight. lol  All fixed.